lib/mix_unused/analyzers/unreachable/usages/phoenix_discovery.ex

```diff
@@ -25,45 +25,50 @@ defmodule MixUnused.Analyzers.Unreachable.Usages.PhoenixDiscovery do
   Note: generally plugs are defined inside pipelines, but the discovery module currently does not check if a pipeline is actually used.
   """
 
+  alias MixUnused.Analyzers.Unreachable.Usages.Context
   alias MixUnused.Analyzers.Unreachable.Usages.Helpers.Aliases
   alias MixUnused.Analyzers.Unreachable.Usages.Helpers.Source
 
   @behaviour MixUnused.Analyzers.Unreachable.Usages
 
   @http_methods [
-    :get,
+    :delete,
```

Comment:
Here we could probably also add `:connect, :head, :trace` for completeness, covering all the HTTP verbs exposed by `Phoenix`

Reply:
Oh yes thanks!